CleanSwift/CleanSwift/DetailScreen/DetailInteractor.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  DetailInteractor.swift
+//  CleanSwift
+//
+//  Created by Валентин Коскин on 17.02.2023.
+//
+
+import Foundation
+
+/// Интерактор экрана деталей
+final class DetailInteractor: DetailBuisnessLogic, DetailDataStore {
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let emptyString = ""
+    }
+    // MARK: - Public Properties
+    
+    var spacePictureDescription: String = Constants.emptyString
```

Comment:
```suggestion
    var spacePictureDescription = Constants.emptyString
```
явное указание сделай в константе